packages/react-component-library/src/components/RangeSlider/Tick.tsx

```diff
@@ -30,17 +76,15 @@ export const Tick: React.FC<TickProps> = ({
   const percent: number = isReversed ? 100 - tick.percent : tick.percent // invert if reversed
   const tickValue: number = (domain[1] / 100) * percent
 
-  const tickClasses = classNames('rn-rangeslider__tick', {
-    'is-active': isActive(values, tickValue),
-    ...useThresholdClasses(percent, thresholds, 'tick'),
-  })
-
   return (
     <div data-testid="rangeslider-tick">
-      <div className={tickClasses} style={{ left: `${tick.percent}%` }} />
+      <StyledTick
+        style={{ left: `${tick.percent}%` }}
```

Comment:
Could we do something to make this part of `StyledTick`?

Reply:
I've slightly changed the naming to make it less confusing but the wrapper div is a requirement.

Reply:
Sorry if this was not clear - I was referring to the `style` as it feels strange that we would have a styled-component and then a `style` prop here.